examples/playground/panel-lite-iframe.js

```diff
@@ -0,0 +1,159 @@
+const welcome=`\
```

Comment:
## Maybe

- [ ] Enable rendering non-Panel components by wrapping the last output in a `pn.panel`?